static/js/scoutingsdk.js

```diff
@@ -4463,4 +4463,76 @@ ${_this.escape(teamNumber)} (Blue ${i + 1})
     _this.hash = (string) => {
         return _this.hexFromBytes(sha256(_this.stringToBytes(string)));
     };
+
+    // Add this method right before the closing brace of the ScoutingAppSDK function
+    _this.showLeaderboardPage = async () => {
+        try {
+            const response = await fetch('/api/scouting/leaderboard');
+            const data = await response.json();
+            
+            if (!data.success) {
+                throw new Error('Failed to fetch leaderboard data');
+            }
```

Comment:
reformat like other fetches in the scouting app and use an html error message for the user (display the error message that the server gives) rather than throwing